README.md

```diff
@@ -114,13 +114,15 @@ For global decorators and parameters you can add them to preview.js inside your
 
 ```jsx
 import {withBackgrounds} from '@storybook/addon-ondevice-backgrounds';
-export const decorators = [withBackgrounds, (Story)=> <View style={{color: 'blue'}}><Story/></View>];
+export const decorators = [withBackgrounds, (Story)=> <View style={{flex: 1, color: 'blue'}}><Story/></View>];
 export const parameters = {
-  backgrounds: [
-    {name: 'plain', value: 'white', default: true},
-    {name: 'warm', value: 'hotpink'},
-    {name: 'cool', value: 'deepskyblue'},
-  ],
+  backgrounds: {
```

Comment:
could you add the default so that it looks like this:

```
  backgrounds: {
    default: 'plain',
    values: [
      { name: 'plain', value: 'white' },
      { name: 'warm', value: 'hotpink' },
      { name: 'cool', value: 'deepskyblue' },
    ],
  },
```

default is now placed at the same level as `values`

Reply:
I will update